FrontEnd/src/components/ProfilePage/FormComponents/AdditionalInfo.js

```diff
@@ -1,39 +1,23 @@
 import css from './FormComponents.module.css';
 import { useState, useEffect } from 'react';
-
+import { useUser, useProfile } from '../../../hooks/';
 import HalfFormField from './FormFields/HalfFormField';
-import TextField from './FormFields/TextField';
+import Loader from '../../loader/Loader';
 
 const LABELS = {
-    'foundationYear': 'Рік заснування',
-    'companySize': 'Розмір компанії',
-    'topClients': 'Топ клієнти',
-    'passedAudit': 'Пройдений аудит',
+    'founded': 'Рік заснування',
 };
 
-const TEXT_AREA_MAX_LENGTH = 1000;
-
 const AdditionalInfo = (props) => {
-    const [user, setUser] = useState(props.user);
+    const { user } = useUser();
+    const { profile: mainProfile, mutate: profileMutate } = useProfile();
+    const [profile, setProfile] = useState(props.profile);
```

Comment:
Should you also pass `setProfile` as props to have consistent state

Reply:
Is it the same `profile` as `mainProfile`?

Reply:
profile is used for form data.. to save data before submitting a request

Reply:
I wrote below that I’m not entirely happy with this implementation, but it didn’t work out better for me